### ISSUE TITLE
crash in iOS9

### ISSUE THREAD

**Issue by libleo** [flutter/flutter#51753]
get it from crash report
only in iOS9, but number is large
flutter version is 1.12.13-hotfix.7

```console
channel: AppStore_23230112
utdid: Xk+Te0cGDcIDAEZPae/9xtnR
os: iPhone OS
osVersion: 9.2
triggeredTime: 2020-02-29 08:26:01.448 +0800
brand: Apple
deviceMode: iPhone8,2
clientIp: 113.57.93.4
carrier: 中国电信
access: WiFi
accessSubtype: Unknown
--- --- --- --- --- --- --- --- --- --- --- --- --- --- --- ---
backtrace:
Incident Identifier: 3DAE029C-E45F-4781-92BF-867B924D2457
CrashReporter Key:   TODO
Hardware Model:      iPhone8,2
Process:         FlyFish [198]
Path:            /var/mobile/Containers/Bundle/Application/D6BB1AA7-C904-4508-BC84-3E10E8E649C9/FlyFish.app/FlyFish
Identifier:      Unknown|com.taobao.flyfish
Version:         5.8.7+ (5.8.7)
Code Type:       ARM-64
Parent Process:  ??? [1]

Date/Time:       2020-02-29 00:26:01 +0000
OS Version:      iPhone OS 9.2 (13C75)
Report Version:  104

Exception Type:  SIGSEGV
Exception Codes: SEGV_ACCERR at 0x0
Crashed Thread:  0

Thread 0 Crashed:
0   Flutter                             0x000000010323ccc4 -[SemanticsObjectContainer initWithSemanticsObject:bridge:] + 150724 (accessibility_bridge.mm:525)
1   Flutter                             0x000000010323be5c -[SemanticsObject accessibilityContainer] + 147036 (accessibility_bridge.mm:0)
2   UIAccessibility                     0x000000018d93c56c 0x18d914000 + 165228
3   UIAccessibility                     0x000000018d93c448 0x18d914000 + 164936
4   Flutter                             0x000000010323a410 -[SemanticsObject initWithBridge:uid:] + 140304 (accessibility_bridge.mm:118)
5   Flutter                             0x000000010323d87c -[FlutterSemanticsObject initWithBridge:uid:] + 153724 (accessibility_bridge.mm:0)
6   Flutter                             0x000000010323e634 flutter::AccessibilityBridge::GetOrCreateObject(int, std::__1::unordered_map<int, flutter::SemanticsNode, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::SemanticsNode> > >&) + 157236 (accessibility_bridge.mm:769)
7   Flutter                             0x00000001032446bc flutter::PlatformViewIOS::UpdateSemantics(std::__1::unordered_map<int, flutter::SemanticsNode, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::SemanticsNode> > >, std::__1::unordered_map<int, flutter::CustomAccessibilityAction, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::CustomAccessibilityAction> > >) + 181948 (accessibility_bridge.mm:0)
8   Flutter                             0x00000001032982a4 std::__1::__function::__func<flutter::Shell::OnEngineUpdateSemantics(std::__1::unordered_map<int, flutter::SemanticsNode, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::SemanticsNode> > >, std::__1::unordered_map<int, flutter::CustomAccessibilityAction, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::CustomAccessibilityAction> > >)::$_31, std::__1::allocator<flutter::Shell::OnEngineUpdateSemantics(std::__1::unordered_map<int, flutter::SemanticsNode, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::SemanticsNode> > >, std::__1::unordered_map<int, flutter::CustomAccessibilityAction, std::__1::hash<int>, std::__1::equal_to<int>, std::__1::allocator<std::__1::pair<int const, flutter::CustomAccessibilityAction> > >)::$_31>, void ()>::operator()() + 524964 (functional:1651)
9   Flutter                             0x0000000103252f58 fml::MessageLoopImpl::FlushTasks(fml::FlushType) + 241496 (message_loop_impl.cc:125)
10  Flutter                             0x00000001032551dc fml::MessageLoopDarwin::OnTimerFire(__CFRunLoopTimer*, fml::MessageLoopDarwin*) + 250332 (message_loop_darwin.mm:76)
11  CoreFoundation                      0x00000001837215f4 0x183644000 + 906740
12  CoreFoundation                      0x0000000183721298 0x183644000 + 905880
13  CoreFoundation                      0x000000018371e9ac 0x183644000 + 895404
14  CoreFoundation                      0x000000018364d680 0x183644000 + 38528
15  GraphicsServices                    0x0000000184b5c088 0x184b50000 + 49288
16  UIKit                               0x00000001884c4d90 0x188448000 + 511376
17  FlyFish                             0x00000001003293fc 0x100084000 + 2774012
18  libdyld.dylib                       0x00000001831ee8b8 0x1831ec000 + 10424
```

**Comment by iapicca** [comment]
Hi @libleo 
could  you please provide your `flutter doctor -v` , 
your `flutter run --release --verbose` running on iOS 9,
your `flutter build ios --verbose`
and your `pubspec.yaml` 
Thank you

**Comment by libleo** [comment]
> Hi @libleo
> could you please provide your `flutter doctor -v` ,
> your `flutter run --release --verbose` running on iOS 9,
> your `flutter build ios --verbose`
> and your `pubspec.yaml`
> Thank you
```
[✓] Flutter (Channel unknown, v1.12.13+hotfix.7, on Mac OS X 10.14.6 18G2022, locale zh-Hans-CN)
    • Flutter version 1.12.13+hotfix.7 at /Users/leo/Developer/flutter
    • Framework revision 0ec3a121bc (3 weeks ago), 2020-02-26 14:28:00 +0800
    • Engine revision e1e6ced81d
    • Dart version 2.7.0

 
[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /Users/leo/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-28, build-tools 28.0.3
    • ANDROID_HOME = /Users/leo/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses

[✓] Xcode - develop for iOS and macOS (Xcode 11.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C504
    • CocoaPods version 1.8.4

[✓] Android Studio (version 3.6)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin version 43.0.2
    • Dart plugin version 192.7761
    • Java version OpenJDK Runtime Environment (build 1.8.0_212-release-1586-b4-5784211)

[✓] IntelliJ IDEA Ultimate Edition (version 2019.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA.app
    • Flutter plugin version 38.2.4
    • Dart plugin version 192.6262.58

[✓] IntelliJ IDEA Community Edition (version 2019.2.1)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 38.2.4
    • Dart plugin version 192.6262.58

[!] VS Code (version 1.36.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    ✗ Flutter extension not installed; install from
      https://marketplace.visualstudio.com/items?itemName=Dart-Code.flutter

[✓] Connected device (1 available)
    • iPhone6-iOS10.3 • 985E5233-0F5A-490C-BE56-92F5A82EE49A • ios • com.apple.CoreSimulator.SimRuntime.iOS-10-3 (simulator)

! Doctor found issues in 2 categories.
```

it is from crash report. I don't have device

**Comment by libleo** [comment]
> Hi @libleo
> could you please provide your `flutter doctor -v` ,
> your `flutter run --release --verbose` running on iOS 9,
> your `flutter build ios --verbose`
> and your `pubspec.yaml`
> Thank you

pubspec.yaml
```
name: flyfish_flutter
description: A new flutter module project.

# The following defines the version and build number for your application.
# A version number is three numbers separated by dots, like 1.2.43
# followed by an optional build number separated by a +.
# Both the version and the builder number may be overridden in flutter
# build by specifying --build-name and --build-number, respectively.
# In Android, build-name is used as versionName while build-number used as versionCode.
# Read more about Android versioning at https://developer.android.com/studio/publish/versioning
# In iOS, build-name is used as CFBundleShortVersionString while build-number used as CFBundleVersion.
# Read more about iOS versioning at
# https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
version: 1.0.0+1

environment:
  sdk: ">=2.1.0 <3.0.0"

dependencies:
  flutter:
    sdk: flutter

  # The following adds the Cupertino Icons font to your application.
  # Use with the CupertinoIcons class for iOS style icons.
  cupertino_icons: ^0.1.2
  transparent_image: ^0.1.0
  cached_network_image: ^2.0.0
  json_annotation: ^2.0.0
  sticky_headers: ^0.1.8
  pulltorefresh_flutter: ^0.1.9
  shared_preferences: 0.4.3
  pull_to_refresh: ^1.5.4
  intl: ^0.16.0
  fluttertoast: ^3.1.3
  common_utils: ^1.1.3
  photo_view: ^0.4.2
  badges: ^1.1.0
  flutter_staggered_grid_view: ^0.3.0

dev_dependencies:
  flutter_test:
    sdk: flutter
  build_runner: ^1.0.0
  json_serializable: ^3.1.0



# For information on the generic Dart part of this file, see the
# following page: https://dart.dev/tools/pub/pubspec

# The following section is specific to Flutter.
flutter:

  # The following line ensures that the Material Icons font is
  # included with your application, so that you can use the icons in
  # the material Icons class.
  uses-material-design: true

  assets:


  # To add assets to your application, add an assets section, like this:
  # assets:
  #  - images/a_dot_burr.jpeg
  #  - images/a_dot_ham.jpeg

  # An image asset can refer to one or more resolution-specific "variants", see
  # https://flutter.dev/assets-and-images/#resolution-aware.

  # For details regarding adding assets from package dependencies, see
  # https://flutter.dev/assets-and-images/#from-packages

  # To add custom fonts to your application, add a fonts section here,
  # in this "flutter" section. Each entry in this list should have a
  # "family" key with the font family name, and a "fonts" key with a
  # list giving the asset and other descriptors for the font. For
  # example:
  # fonts:
  #   - family: Schyler
  #     fonts:
  #       - asset: fonts/Schyler-Regular.ttf
  #       - asset: fonts/Schyler-Italic.ttf
  #         style: italic
  #   - family: Trajan Pro
  #     fonts:
  #       - asset: fonts/TrajanPro.ttf
  #       - asset: fonts/TrajanPro_Bold.ttf
  #         weight: 700
  #
  # For details regarding fonts from package dependencies,
  # see https://flutter.dev/custom-fonts/#from-packages

  #
  # This section identifies your Flutter project as a module meant for
  # embedding in a native host app.  These identifiers should _not_ ordinarily
  # be changed after generation - they are used to ensure that the tooling can
  # maintain consistency when adding or modifying assets and plugins.
  # They also do not have any bearing on your native host application's
  # identifiers, which may be completely independent or the same as these.
  # 
  module:
    androidPackage: com.taobao.flyfish_flutter
    iosBundleIdentifier: com.taobao.flyfishFlutter
```

**Comment by iapicca** [comment]
Thank you @libleo

**Comment by chinmaygarde** [comment]
@jason-simmons: This seems like something that you worked on and fixed on ToT. I am not able to recall the commit though. v1.12.13 is a pretty old release.
